### PR TITLE
Run Jupyter image and centraldashboard image release on postsubmit

### PR DESCRIPTION
**Pull request by jlewi**
* The autoimage releaser cron job is broken because the recently added centraldashboard
  workflow is using ksonnet app version 0.2.0 (which requires ksonnet 0.12) while
  the other release workflows are using version 0.1.0 of the APi (ksonnet 0.11)

* The simplest fix is to use 0.1.0.

* To make this easier consolidate all the release workflows in a single
  ksonnet app.

Related to #1563

* Trigger the centraldashboard image release workflow on presubmit.
  Related to #1574 (Use prow to manage image release workflows.)

  * Shorten the dashboard name to avoid hitting pod naming limits.

* Set CPU and RAM on the containers in release.libsonnet to see if this speeds up building
  the docker images.

* Add building of Jupyter Docker images to pre and postsubmits so we verify this is working. This will also give us something to compare against w.r.t the central dashboard images.

* Disable image builds on presubmits until they are passing.

Related to #1576 Jupyter image builds failing
Related to #1575 Central dashboard image builds aren't working.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1565)

### CODE REVIEW

File: prow_config.yaml [hunk 1]
```diff
@@ -52,3 +52,45 @@ workflows:
...
+
+  # Image Auto Release workflows.
+  # The workflows below are related to auto building our Docker images.
+  # We have separate pre/postsubmit jobs because we want to use different
+  # registries
+  - app_dir: kubeflow/kubeflow/releasing/releaser
+    component: centraldashboard
+    name: dashboard-release
+    job_types:
+      - presubmit
```
Comment (kunmingg) [inline prow_config.yaml]: Maybe we want to run release workflows only on postsubmit.

File: prow_config.yaml [hunk 2]
```diff
@@ -52,3 +52,45 @@ workflows:
       - kubeflow/tf-serving/*
       - components/k8s-model-server/*
       - testing/*
+
+  # Image Auto Release workflows.
+  # The workflows below are related to auto building our Docker images.
```
Comment (kunmingg) [inline prow_config.yaml]: Should we delete the current release cron job if prow will take over?

Comment (jlewi) [reply prow_config.yaml]: We should but lets do that in a follow on PR.

**Pull request merged**